proto/time.proto

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreedto in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package contains a shared time data structure
+
+syntax = "proto3";
+option go_package = "vitess.io/vitess/go/vt/proto/vttime";
+
+package vttime;
+
+// Time represents a time stamp in nanoseconds. In go, use logutil library
+// to convert times.
+message Time {
+  int64 seconds = 1;
+  int32 nanoseconds = 2;
```

Comment:
I'd say go with `int64`. There's no great benefit to using `int32`.

Reply:
this is an existing message that I moved into a new file. Is it safe to change the data type?

Reply:
we'll keep this as int32 because changing the type will not be backwards-compatible